elisp/log4slime.el

```diff
@@ -1159,7 +1156,7 @@ menu, by customizing `log4slime-menu-levels' variable`
 
               (unless (eq expert 'expert)
                 (let* ((window-min-height 2)
-                       (window-tree))
```

Comment:
This change seems to be incomplete as byte-compiling stills results in
```
log4slime.el:1281:20: Warning: reference to free variable ‘window’
log4slime.el:1281:20: Warning: assignment to free variable ‘window’
```
I think this is because further down there are more uses of `window` that are not within the lexical scope of this binding.

Reply:
Yes, and the references need to be in the same lexical environment.

Thanks.  I think what I have checked in is correct now.

Reply:
Thank you, that fixes the warning. Would it be possible to do the following now?
```diff
diff --git a/elisp/log4slime.el b/elisp/log4slime.el
index 082e9cc..a7169a4 100644
--- a/elisp/log4slime.el
+++ b/elisp/log4slime.el
@@ -1156,8 +1156,7 @@ menu, by customizing `log4slime-menu-levels' variable`
                 (setq mode-line-format nil))
 
               (unless (eq expert 'expert)
-                (let* ((window-min-height 2)
-                       (window-tree))
+                (let ((window-min-height 2))
                   ;; (log-expr nrows (window-height)
                   ;;           (- (window-height) 1 nrows))
```
Since `window-tree` is (now?) unused and the `let*` is reduced to a single binding.

Reply:
Good suggestion.  I have committed that change.

Thanks.